### DISCUSSION TITLE
Postcss Global Data - glob support for files config option.

### DISCUSSION THREAD

**Discussion by dysphasia** [csstools/postcss-plugins#1379] [Ideas]
The README doesn't mention it, and attempting to include a glob path resulted in an error, so I assume it's not supported. It seems like a useful addition though.

```js
    postcssGlobalData({
      files: [
        './src/styles/mixins/*.{css,pcss}',
        './src/styles/vars/*.{css,pcss}'
      ]
    })
```

**Comment by romainmenke** [comment]
That seems like something that can be done with any glob dependency in a javascript config file?
Not something that needs to be supported in the plugin itself.

Right?

**Reply by dysphasia** [reply]
Do you mean like:

```js
// postcss.config.js

import postcssGlobalData from '@csstools/postcss-global-data';
import { glob } from 'glob';

const globalPaths = await glob('./src/styles/{mixins,vars}/*.{css,pcss}');

export default {
  plugins: [
    postcssGlobalData({
      files: globalPaths
    })
  ]
}
```

This works. Supporting it in the plugin would just be for convenience. I understand the philosphy though. Thank you for the suggestion.

**Reply by romainmenke** [reply]
Yup exactly like that :)
